chapters/lexicalstructure.tex

```diff
@@ -82,7 +82,7 @@ \subsection{Identifiers}\doublelabel{identifiers}
 \lstinline!'+foo'!. Control characters in quoted identifiers have to use string
 escapes.
 The single quotes are part of the identifier, i.e., \lstinline!'x'! and \lstinline!x!
-are distinct identifiers, but the redundant escapes (\lstinline!'\?'! and \lstinline!'\"'!) are the same as the corresponding non-escaped
+are distinct identifiers. But the redundant escapes (\lstinline!'\?'! and \lstinline!'\"'!) are the same as the corresponding non-escaped
 variants (\lstinline!'?'! and \lstinline!'"'!), and outside of Modelica source code the redundant escapes shall not be used.
```

Comment:
> … but are only for use in Modelica source code (excluding, for example, variable names in result files and class names in _package.order_ files).

Reply:
I would prefer to not list the excluded cases, since it is so easy to get carried away.

Reply:
Sure, but it would still be a small improvement to use a shorter 'but' formulation:
> … but are only for use in Modelica source code.

Reply:
Ah, yes.